source/frontend/src/components/common/form/Multiselect.scss

```diff
@@ -7,3 +7,16 @@
 .highlightOption {
   background: #1967d2 !important;
 }
+
+.search-wrapper {
+  padding-top: 0rem !important;
+  padding-bottom: 0rem !important;
+}
+
+.search-wrapper input {
+  margin-top: 5px !important;
+}
+
+.chip {
+  margin-top: 5px;
```

Comment:
we try to use **rem** instead of px in new CSS code - please change to **0.5rem** (1rem = 10px)